lib/main.coffee

```diff
@@ -296,36 +288,49 @@ module.exports = Hydrogen =
             @moveDown endRow
 
     removeStatusBarElement: ->
-        if @statusBarElement?
-            while @statusBarElement.hasChildNodes()
-                @statusBarElement.removeChild(@statusBarElement.lastChild)
-
-    setStatusBarElement: (element) ->
-        if @statusBarElement?
-            @removeStatusBarElement()
-            @statusBarElement.appendChild(element)
-        else
-            console.error "No status bar element. Can't set it."
+        unless @statusBarElement?
+            console.error 'removeStatusBarElement: there is no status bar'
+            return
+
+        while @statusBarElement.hasChildNodes()
+            @statusBarElement.removeChild @statusBarElement.lastChild
+
+    setStatusBarElement: ->
+        unless @statusBarElement?
+            console.error 'setStatusBarElement: there is no status bar'
```

Comment:
This line throws an error every time Hydrogen is activated. This didn't happen before.